addon/utils/cookies.js

```diff
@@ -0,0 +1,31 @@
+let Cookies = {
+  create(name, value, days) {
+    var expires;
+
+    if (days) {
+      var date = new Date();
+      date.setTime(date.getTime() + (days * 24 * 60 * 60 * 1000));
+      expires = "; expires=" + date.toGMTString();
+    } else {
+      expires = "";
+    }
+    document.cookie = encodeURIComponent(name) + "=" + encodeURIComponent(value) + expires + "; path=/";
```

Comment:
Does a domain need to be specified?
Is there a missing "; " between the encoded value and the `expires`?